src/web/mjs/connectors/Kuman5.mjs

```diff
@@ -1,21 +1,14 @@
-import Connector from '../engine/Connector.mjs';
-import Manga from '../engine/Manga.mjs';
+import MH from './templates/MH.mjs';
 
-export default class Kuman5 extends Connector {
+export default class kuman5 extends MH {
     constructor() {
         super();
         super.id = 'kuman5';
         super.label = '酷漫屋 (Kuman55)';
         this.tags = [ 'manga', 'chinese' ];
         this.url = 'http://www.kuman55.com';
-    }
```

Comment:
In case of the placeholder solution, overwrite the path, e.g.
`this.path = '/sort/1-%PAGE%.html';`